### DISCUSSION TITLE
List of all admin commands with "!admin" (of course only available to admins)

### DISCUSSION THREAD

**Discussion by Albert-Bruun-Thomsen** [Refloow/Steam-Card-Bot-PRO#51] [Ideas]
List of all admin commands with "!admin" (of course only available to admins)

**Comment by Refloow** [comment]
This resource might answer your discussion topic
https://github.com/Refloow/Steam-Card-Bot-PRO/blob/master/.github/FEATURES.md
or to be more precise
https://github.com/Refloow/Steam-Card-Bot-PRO/blob/master/.github/FEATURES.md#--------------------------admin-commands------------------------

**Reply by Albert-Bruun-Thomsen** [reply]
I thought it more as a quick access so that you don't have to open up the github repo
